Dockerfile.assisted-swarm-build

```diff
@@ -0,0 +1,15 @@
+FROM registry.ci.openshift.org/openshift/release:golang-1.17
+ENV GO111MODULE=on
+ENV GOFLAGS=""
+
+COPY --from=quay.io/goswagger/swagger:v0.28.0 /usr/bin/swagger /usr/bin/goswagger
+COPY --from=quay.io/edge-infrastructure/swagger-codegen-cli:2.4.18 /opt/swagger-codegen-cli /opt/swagger-codegen-cli
+
+USER 0
+
+RUN curl -L https://github.com/stoplightio/spectral/releases/download/v5.9.1/spectral-linux -o /usr/local/bin/spectral && chmod +x /usr/local/bin/spectral
+
+RUN ulimit -n 20000 && yum install -y --setopt=skip_missing_names_on_install=False docker podman
```

Comment:
I'm failing to get this `ulimit` command to work locally during the build

Reply:
Will remove. For me it caused installation to be much faster.